### ISSUE TITLE
flake8-black incorrectly flagging code as "Black would make changes"

### ISSUE THREAD

**Issue by itsthejoker** [peterjc/flake8-black#28]
I have a line that ends with a comma that Black puts there, but flake8-black disagrees. The line in question:
```python
        result = client.get(
            reverse("transcription-search"), content_type="application/json", **headers,
        )
```
Removing the comma at the end will make flake8-black happy, but Black put it there in the first place. By manually removing all of these final commas, flake8-black will allow it to pass.

Running `black --check` reports that the file is fine: 

```sh
black --check api/tests/test_transcriptions.py                           130ms  Mon 07 Sep 2020 02:06:53 PM EDT
All done! ✨ 🍰 ✨
1 file would be left unchanged.
```

**Comment by peterjc** [comment]
Very strange - is it possible you have more than one copy of black present (e.g more than one python) since that thing with commas reminds me of the kind of tiny change that happened between versions of black.

If that doesn't solve it, can you share a minimal self contained example?

**Comment by itsthejoker** [comment]
...that's entirely possible -- pre-commit uses its own venv. Thanks for the lead, I'll take a look and update.

**Comment by peterjc** [comment]
If you are using pre-commit, then yes - that is most likely what is going on. You can try this too:

```
$ flake8 --select BLK api/tests/test_transcriptions.py
```

vs:

```
$ black --check api/tests/test_transcriptions.py
```

**Comment by peterjc** [comment]
And to rule out different versions of Python:

```
$ python -m flake8 --select BLK api/tests/test_transcriptions.py
```

vs:

```
$ python -m black --check api/tests/test_transcriptions.py
```

**Comment by peterjc** [comment]
As an aside, I would suggest you use black directly in pre-direct rather than using flake8-black - you'll find it faster thanks to black's cache #26. However, you may find that took many hooks makes the pre-commit tool too verbose.

**Comment by itsthejoker** [comment]
Confirmed that black was different versions between pre-commit and the primary project. Thanks so much for your help!!
